doc/source/build_hints.rst

```diff
@@ -1803,11 +1803,68 @@ The following options are available to select a subset of drivers:
     Independently of options that control global behavior, drivers can be individually
     enabled or disabled with those options.
 
+    .. note::
+
+        <driver_name> above and below is *generally*, but not systematically the short driver name.
+
+        Some drivers may also be grouped together for build purposes.
+
+        - A number of "raw" raster drivers (ACE2, BT, BYN, CPG, CTable2, DIPX, DOQ1,
+          DOQ2, EHDR, EIR, ENVI, FAST, GenBIN, GSC, GTX, MFF2, ISCE, KRO, MFF, LAN,
+          LCP, LOSLAS, NDF, NTv2, PAUX, PNM, ROIPAC, RRASTER, SNODAS) are controlled
+          by the GDAL_ENABLE_DRIVER_RAW option.
+
+        - Planetary raster formats (PDS, PDS4, ISIS, ISIS3, VICAR) are controlled by
```

Comment:
Should this be ISIS2 or should ISIS2 in https://gdal.org/drivers/raster/index.html be renamed to ISIS?

Reply:
yes this is ISIS2. fixing